app/locale/af/LC_MESSAGES/django.po

```diff
@@ -812,6 +812,12 @@ msgid ""
 "NTT."
 msgstr "Vir fone benewens selfone, gebruik die 171-diens van NTT."
 
+msgid "For responders"
```

Comment:
It's here:
https://github.com/google/personfinder/blob/d000d626fc2bcb14b54e7cec5790add90e938bde/app/resources/static-base.html.template#L67
though it's been already there, not introduced in this pull request. I believe I forgot to run update_message script when I added the message.